maze-info.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/activity/activity.info b/activity/activity.info
+index abb1d02..76aba63 100644
+--- a/activity/activity.info
++++ b/activity/activity.info
+@@ -3,9 +3,14 @@ name = Maze
+ exec = sugar-activity3 activity.MazeActivity
+ icon = activity-maze
+ activity_version = 29
++release_date = 2018-04-12
+ license = GPLv3+
++metadata_license = CC0-1.0
++update_contact = tch@sugarlabs.org
+ bundle_id = vu.lux.olpc.Maze
+ summary = Good at finding your way out of hard situations? Let's see! Try to make your way through an increasingly difficult path, or you can also play with a friend!
+ max_participants = 6
+ repository = https://github.com/sugarlabs/maze-activity
```

Comment:
We need an `url` field as well, maybe it can point to the wiki page for now, or we can add something quick to the help.sugarlabs.org/en/ web page.

Reply:
<img width="795" alt="image" src="https://user-images.githubusercontent.com/30125762/71457383-a9aea380-276b-11ea-8872-5205994c2833.png">
i dont see any wiki page

Reply:
3a859ce i just made the url the github repo for now